components/inputs/test/input-number.test.js

```diff
@@ -366,6 +367,20 @@ describe('d2l-input-number', () => {
 
 			expect(fired).to.be.false;
 		});
+
+		it('should set value to undefined', async() => {
```

Comment:
Can't get the tests to line up correctly with the fix yet. Testing in engagement-dashboard shows me the correct behaviour now (blank instead of zero), but I can't get the tests set up to match that behaviour 😢

Reply:
Not sure if you've tried this: reach into the inner `<input type="text">` and blank out the value, then trigger a `change` event originating from it -- essentially trying to simulate the user clearing it.